sway-lib-std/src/array_conversions/u16.sw

```diff
@@ -84,11 +84,11 @@ impl u16 {
         let output = [0_u8, 0_u8];
 
         asm(input: self, off: 0xFF, i: 0x8, output: output, r1) {
-            srl r1 input i;
-            sb output r1 i0;
+            srl  r1 input i;
+            sb   output r1 i0;
```

Comment:
I feel like we either need to align every column perfectly or just go with a single space between each token.